CRM/Activity/BAO/Activity.php

```diff
@@ -2030,8 +2030,8 @@ public static function getContactActivity($contactId) {
 
     $dao = CRM_Core_DAO::executeQuery($query);
 
-    $activityTypes = CRM_Core_OptionGroup::values('activity_type');
-    $activityStatuses = CRM_Core_OptionGroup::values('activity_status');
+    $activityTypes = CRM_Activity_BAO_Activity::buildOptions('activity_type_id', 'get');
+    $activityStatuses = CRM_Activity_BAO_Activity::buildOptions('activity_status_id', 'get');
```

Comment:
yes, this looks correct, being used for rendering a label within the results. CRM_Core_Pseudoconstant::getLabel would make even more sense in the function context

Reply:
@eileenmcnaughton As it's trivial and neater to use getLabel here I've updated the PR to do so!